src/PointwiseFunctions/AnalyticSolutions/Poisson/Lorentzian.hpp

```diff
@@ -0,0 +1,75 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"     // IWYU pragma: keep
+#include "Elliptic/Systems/Poisson/Tags.hpp"    // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Poisson {
+namespace Solutions {
+
+/*!
+ * \brief A Lorentzian solution to the Poisson equation
+ *
+ * \details This implements the Lorentzian solution
+ * \f$u(\boldsymbol{x})=\left(1+r^2\right)^{-\frac{1}{2}}\f$ to the
+ * three-dimensional Poisson equation
+ * \f$-\Delta u(\boldsymbol{x})=f(\boldsymbol{x})\f$, where
+ * \f$r^2=x^2+y^2+z^2\f$. The corresponding source is
+ * \f$f(\boldsymbol{x})=3\left(1+r^2\right)^{-\frac{5}{2}}\f$.
+ *
+ * \note Corresponding 1D and 2D solutions are not implemented yet.
+ */
+template <size_t Dim>
```

Comment:
I don't really see a need to template this.  If you want to I'm OK with it, but add a static_assert that Dim is 3.